crates/rspack_napi/src/threadsafe_function/mod.rs

```diff
@@ -67,7 +80,7 @@ impl<T: 'static, R> ThreadsafeFunction<T, R> {
   }
 
   fn call_with_return<D: 'static + FromNapiValue>(&self, value: T) -> Receiver<Result<D>> {
-    let (tx, rx) = tokio::sync::oneshot::channel::<Result<D>>();
+    let (tx, rx) = oneshot::channel::<Result<D>>();
```

Comment:
[`tokio::sync::oneshot::Receiver::blocking_recv`](https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.Receiver.html#method.blocking_recv) panics when called within an async context.

Replacing it with [`onehost::Receiver::recv`](https://docs.rs/oneshot/latest/oneshot/struct.Receiver.html#method.recv).